cpp/ql/test/library-tests/dataflow/DefaultTaintTracking/annotate_sinks_only/defaulttainttracking.cpp

```diff
@@ -156,35 +117,35 @@ void sink(Point);
 void test_field_to_obj_taint_object(Point p) {
   p.x = getenv("VAR")[0];
   sink(p); // not tainted
-  sink(p.x); // tainted
+  sink(p.x); // $ ir MISSING: ast
 }
 
 void test_field_to_obj_taint_object_addrof(Point p) {
   taint_x(&p);
-  sink(p); // tainted [field -> object]
-  sink(&p); // tainted [field -> object]
-  sink(p.x); // tainted
+  sink(p); // $ MISSING: ast,ir // tainted [field -> object]
```

Comment:
Oh yeah. The lines with `tainted [field -> object]` comments are actually supposed to missing. They are examples of flow from field to object (which we intend to remove as part of https://github.com/github/codeql-c-analysis-team/issues/103), and https://github.com/github/codeql/pull/4230 removed these results (but apparently I didn't update the annotations, sorry!).

Reply:
Ah, good to know, I'll remove all the MISSING annotations then.